code/game/objects/effects/decals/cleanable/misc.dm

```diff
@@ -9,9 +9,11 @@
 	desc = "Ashes to ashes, dust to dust, and into space."
 	icon = 'icons/obj/objects.dmi'
 	icon_state = "ash"
-	mergeable_decal = FALSE
+	mergeable_decal = TRUE
 
-/obj/effect/decal/cleanable/ash/Initialize()
+/obj/effect/decal/cleanable/ash/Initialize(disable_turf_loc_check=FALSE)

```

Comment:
`Initialize(disable_turf_loc_check=FALSE)` the first argument to Initialize is always mapload, this causes maploading ash to override turf checks, but also makes check-disabled instantiation behave as if it was maploading. Please extend on the signature of the parent type, which is `/obj/effect/decal/cleanable/Initialize(mapload, list/datum/disease/diseases)`, to end up with `/obj/effect/decal/cleanable/ash/Initialize(mapload, list/datum/disease/diseases, disable_turf_loc_check=FALSE)`

Reply:
An alternate way would be to make the signature `/obj/effect/decal/cleanable/ash/Initialize(mapload, disable_turf_loc_check=FALSE)` and then call the parent with explicit arguments like `..(mapload)`